gitkraken-client/single-sign-on.md

```diff
@@ -291,4 +291,92 @@ Step 10: Select “I am an Okta customer adding an internal app” from option m
 
 <img src="/wp-content/uploads/sso-otka-7.png" class="img-bordered img-responsive center">
 
-Now you are all set to [setup your SSO on a GitKraken Organization](/gitkraken-client/single-sign-on/#setting-up-sso-on-a-gitKraken-organization)
\ No newline at end of file
+Now you are all set to [setup your SSO on a GitKraken Organization](/gitkraken-client/single-sign-on/#setting-up-sso-on-a-gitKraken-organization)
+
+### Ping Identity
+
+How to Create SAML Application in Ping Identity
+
+1. Go to https://www.pingidentity.com/en/account/sign-on.html
+
+2. Create an account or sign in your existing one.
+
+3. Go to the home page and click on *Add Environment*.
+
+<img src="/wp-content/uploads/sso-pingidentity-3.png" class="img-bordered img-responsive center">
+
+4. Select the appropriate solution based on your need (in this guide, we use *Customer solution*) and click *Next*.
+
+<img src="/wp-content/uploads/sso-pingidentity-4.png" class="img-bordered img-responsive center">
+
+5.	Select *PingOne SSO*, then click *Next*.
+
+<img src="/wp-content/uploads/sso-pingidentity-5.png" class="img-bordered img-responsive center">
+
+6.	Type in your environment name (in *TRIAL ENVIRONMENT NAME*), then click “Finish”. Now your environment is created. Go ahead and click on it.
+
+<img src="/wp-content/uploads/sso-pingidentity-6.png" class="img-bordered img-responsive center">
+<img src="/wp-content/uploads/sso-pingidentity-6-2.png" class="img-bordered img-responsive center">
+
+7.	Select *Identities* to add some users. Once you are done adding them, go ahead and click on *Groups*, then click on the plus button to add a group. (Make sure to add users with their email addresses).
```

Comment:
Remove unneeded words from this sentence.

Select *Identities* to add some users. Once you are done adding them~, go ahead and~ click on *Groups*, then click on the plus button to add a group. (Make sure to add users with their email addresses).